tools/gentool/gentool.go

```diff
@@ -4,8 +4,10 @@ import (
 	"flag"
 	"fmt"
 	"log"
+	"os"
 	"strings"
 
+	"gopkg.in/yaml.v2"
```

Comment:
Why not using `yaml.v3`?

Reply:
I think you use 2, so you keep it.
Let me update it.